scalameta/parsers/shared/src/main/scala/scala/meta/internal/parsers/ScalametaParser.scala

```diff
@@ -4181,8 +4181,8 @@ class ScalametaParser(input: Input)(implicit dialect: Dialect) { parser =>
         else
           in.observeIndented()
 
-      if (nextIndented)
```

Comment:
the nextIndented variable is no longer used.

Reply:
Oops, right, thanks!